tests/OSPSuite.Presentation.Tests/Importer/Presenters/ImporterPresenterSpecs.cs

```diff
@@ -65,6 +88,28 @@ protected override void Context()
             moleculeMetaDataCategory,
             createMetaDataCategory<string>("Mol weight", isMandatory: false)
          };
+         var dataFormat = A.Fake<IDataFormat>();
+         A.CallTo(() => dataFormat.Parameters).Returns(new List<DataFormatParameter>());
+         var dataSourceFile = A.Fake<IDataSourceFile>();
+         A.CallTo(() => dataSourceFile.Format).Returns(dataFormat);
+         var importerPresenter = A.Fake<IImporterDataPresenter>();
+         A.CallTo(() => importerPresenter.SetDataSource(A<string>.Ignored)).Returns(dataSourceFile);
+         sut = new ImporterPresenterForTest(
```

Comment:
The fact that you are creating fake without holding references is a clear sign that we have an implementation issue or that the tests are not testing the behavior of your sut. I think we should review this in another issue

Reply:
I am not sure I understand your point here, but let's actually make a new issue out of it and discuss this in detail